gramineproject/gramine
# Convert manifests to new mount format

## Description of the changes 

Follow-up to #411. Converts remaining manifests and documentation.

For the manifests, I used editor automation (regexp replace + a macro), but some manual adjustments were also necessary.



## How to test this PR? 

Read it carefully.

Here are some checks I did to find what to convert:
* `git grep 'fs\.mount\.'` - old table syntax
* `git grep 'path = "[^/]'` - non-absolute paths (but also shows paths with variables like `"{{ arch_libdir }}/..."`)
* `git grep 'type = "chroot"'` - should not be necessary anymore


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/gramineproject/gramine/417)
